core/strings/builder.odin

```diff
@@ -46,6 +46,14 @@ builder_cap :: proc(b: Builder) -> int {
 	return cap(b.buf);
 }
 
+builder_pop_rune :: proc(b: ^strings.Builder) -> (rune, int) {
+	r, size := utf8.decode_last_rune_in_string(string(b.buf[:]));
+	raw_dyn := transmute(^mem.Raw_Dynamic_Array)&b.buf;
```

Comment:
This can indeed just be a `cast(^mem.Raw_Dynamic_Array)` since it's just a pointer.